spec/domain/content/query_spec.rb

```diff
@@ -143,8 +144,31 @@
     let!(:content_item_3) { create(:content_item, six_months_page_views: 1) }
     let!(:content_item_4) { create(:content_item, six_months_page_views: 9) }
 
-    it "defaults to six month page views descending" do
-      expect(subject.content_items.pluck(:six_months_page_views)).to match_array [9, 3, 3, 1]
+    describe "with default sort" do
+      it "sorts by six month page views descending" do
+        expect(subject.content_items).to contain_exactly(
+          content_item_4,
+          content_item_2,
+          content_item_1,
+          content_item_3,
+        )
+      end
+
+      it "can pick items after another item" do
+        subject.after(content_item_1)
+        expect(subject.content_items).to contain_exactly(content_item_3)
+      end
+    end
+
+    describe "sorting by page views ascending" do
+      before do
+        subject.sort_direction(:asc)
+      end
+
+      it "can pick items after another item" do
```

Comment:
Could you please isolate in a `describe` the tests for `after`.
It would be good to test what happens when the element is the last position, last position of a page, first position, ..., in the middle. We have had so many issues with this feature, that it would be good to describe all the edge cases.